package.json

```diff
@@ -22,7 +22,21 @@
     "url": "https://github.com/internetarchive/bookreader/issues"
   },
   "homepage": "https://github.com/internetarchive/bookreader#readme",
-  "dependencies": {},
+  "dependencies": {
+    "@internetarchive/ia-book-downloads": "^0.1.5",
```

Comment:
Don't you need the dependencies to be here to get them to be included with the built package?

Reply:
Yes and no-ish? Currently BR compiles into single JS files which are imported in petabox as `<script>` tags. Longer term we likely want to change that so that it can be imported like a more traditional npm repo, but we currently don't do that.

Reply:
updated